x/fswap/spec/01_concepts.md

```diff
@@ -0,0 +1,43 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Swap
```

Comment:
Providing detailed information would be appreciated:

1. Who can trigger a swap, by which message.
2. Abstract behavior about swap (burn or lock, etc).
3. descriptions about the fields in the messages or references to protobuf so one easily access the description.